### PR TITLE
Null is a bad variable name

### PR DESCRIPTION
**Pull request by adamsong**
# Document the changes in your pull request

This doesn't make that parameter null, it makes a parameter named null, which is probably not intended behavior

### CODE REVIEW

File: code/game/mecha/mecha_wreckage.dm [hunk 1]
```diff
@@ -80,7 +80,7 @@
 			to_chat(user, span_warning("You don't see anything that can be pried with [I]!"))
 
 
-/obj/structure/mecha_wreckage/transfer_ai(interaction, mob/user, null, obj/item/aicard/card)
+/obj/structure/mecha_wreckage/transfer_ai(interaction, mob/user, _, obj/item/aicard/card)
```
Comment (ToasterBiome) [inline code/game/mecha/mecha_wreckage.dm]:
```suggestion
/obj/structure/mecha_wreckage/transfer_ai(interaction, mob/user, mob/living/silicon/ai/AI, obj/item/aicard/card)
```

even if we're not gonna use it, might as well have it match the actual proc.

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

File: code/game/mecha/mecha_wreckage.dm (hunk 1 above)
Comment (adamsong) [reply code/game/mecha/mecha_wreckage.dm]: That will break the proc, as it references the object level AI variable

Comment (ToasterBiome) [inline code/game/mecha/mecha_wreckage.dm]:
```suggestion
/obj/structure/mecha_wreckage/transfer_ai(interaction, mob/user, mob/living/silicon/ai/ai_unused, obj/item/aicard/card) //ai_unused is unused but having it is better than having  it be null or _ for readability
```

**Review by ToasterBiome — DISMISSED** [review]

**Comment by ToasterBiome** [comment]
https://github.com/yogstation13/Yogstation/pull/13616

Will merge whichever has a less stupid name for the variable soonest

**Review by JamieD1 — APPROVED** [review]

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**